addons/stock/report/report_stock_quantity.py

```diff
@@ -46,19 +46,21 @@ def init(self):
         tools.drop_view_if_exists(self._cr, 'report_stock_quantity')
         query = """
 CREATE or REPLACE VIEW report_stock_quantity AS (
-WITH
+WITH warehouse_cte AS(
+    SELECT sl.id as sl_id, w.id as w_id
+    FROM stock_location sl
+    LEFT JOIN stock_warehouse w ON sl.parent_path::text ~~ concat('%%/', w.view_location_id, '/%%')
```

Comment:
Can you use `LIKE` instead? Had to look up what `~~` meant, and I'm pretty sure I won't be the only one. :sweat_smile: 
Might get easier/quicker to understand overall.
Also, can you indent this cte on the same level as the following ones?